### PR TITLE
Refactor ctor

### PR DESCRIPTION
**Pull request by molnard**

### CODE REVIEW

**Comment by molnard** [comment]
Go for it @nopara73 ! :)

File: Sake/Mixer.cs [hunk 1]
```diff
@@ -1,37 +1,37 @@
...
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+﻿using NBitcoin;
 
 namespace Sake
 {
     internal class Mixer
     {
-        /// <param name="feeRate">Bitcoin network fee rate the coinjoin is targeting.</param>
```
Comment (nopara73) [inline Sake/Mixer.cs:18]: Please explain the new things you added properly and not remove the old ones.

**Comment by molnard** [comment]
Please review this first https://github.com/nopara73/Sake/pull/9

**Comment by nopara73** [comment]
image

**Pull request merged**